### PR TITLE
CONSOLE-4171: i18n upload/download routine task - sprint 256

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
Adding latest translations from Memsource project https://cloud.memsource.com/web/project2/show/K06O0O5XHgFgGk62t3JKLd

### CODE REVIEW

File: frontend/public/locales/zh/public.json [hunk 1]
```diff
@@ -1856,13 +1856,13 @@
   "ClusterRole": "集群角色",
   "ClusterRoles": "集群角色",
   "SelfSubjectAccessReview": "SelfSubjectAccessReview",
-  "SelfSubjectAccessReviews": "SelfSubjectAccessReviews",
+  "SelfSubjectAccessReviews": "selfsubjectaccessreviews",
```
Comment (sg00dwin) [inline frontend/public/locales/zh/public.json:1859]: Should this be CapitalCase?

File: frontend/public/locales/zh/public.json [hunk 2]
```diff
@@ -1856,13 +1856,13 @@
...
   "SelfSubjectReview": "SelfSubjectReview",
   "SelfSubjectReviews": "SelfSubjectReviews",
   "ResourceAccessReview": "ResourceAccessReview",
   "ResourceAccessReviews": "ResourceAccessReviews",
   "LocalResourceAccessReview": "LocalResourceAccessReview",
-  "LocalResourceAccessReviews": "LocalResourceAccessReviews",
+  "LocalResourceAccessReviews": "localresourceaccessreviews",
```
Comment (sg00dwin) [inline frontend/public/locales/zh/public.json:1865]: Should this be CapitalCase?

Comment (cyril-ui-developer) [reply frontend/public/locales/zh/public.json:1865]: > Should this be CapitalCase?

Hmm, not sure why the translators made this change. But I will open a bug for the investigation.

**Pull request merged**